test/QueryPlannerTestHelpers.h

```diff
@@ -259,6 +262,48 @@ inline auto TransitivePath =
                             TransitivePathSideMatcher(right))));
     };
 
+inline auto PathSearchConfigMatcher = [](PathSearchConfiguration config) {
+  auto sourceMatcher =
+      std::holds_alternative<Variable>(config.sources_)
+          ? AD_FIELD(
+                PathSearchConfiguration, sources_,
+                VariantWith<Variable>(Eq(std::get<Variable>(config.sources_))))
+          : AD_FIELD(
+                PathSearchConfiguration, sources_,
+                VariantWith<std::vector<ValueId>>(UnorderedElementsAreArray(
+                    std::get<std::vector<ValueId>>(config.sources_))));
```

Comment:
Can't this just be `AD_FIELD(Config, sources_, Eq(config.sources_))`
?
Same for the `targetMatcher`.